src/test/java/org/padaiyal/utilities/commandline/StreamUtilityTest.java

```diff
@@ -0,0 +1,102 @@
+package org.padaiyal.utilities.commandline;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvFileSource;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.mockito.ArgumentMatchers;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import org.padaiyal.utilities.PropertyUtility;
+
+/**
+ * Tests for StreamUtility.
+ */
+class StreamUtilityTest {
+
+  /**
+   * Tests StreamUtility::convertInputStreamToString() with valid input.
+   *
+   * @param testString    String to test.
```

Comment:
formatting of docstring doesn't match formatting of other classes.

Reply:
Done.